website/server/libs/slack.js

```diff
@@ -33,6 +34,10 @@ function sendFlagNotification ({
   let title = `Flag in ${group.name}`;
   let text = `${flagger.profile.name} (${flagger.id}) flagged a message (language: ${flagger.preferences.language})`;
 
+  if (userComment) {
+    text += `User Commented: ${userComment}`;
```

Comment:
Can you change "User Commented:" to " and commented:"? Note there's a space in front of "and". At the moment it's run on like this:
`LadyAlys (d904bd62-da08-416b-a816-ba797c9ee265) flagged a message (language: en)User Commented: ...`
and we want this:
`LadyAlys (d904bd62-da08-416b-a816-ba797c9ee265) flagged a message (language: en) and commented: ...`